src/screens/FeedScreen/ContentLink.js

```diff
@@ -98,11 +99,12 @@ const ContentLink = ({
 export default ContentLink;
 
 const styles = StyleSheet.create({
-  contentFeed: {
-    flex: 1,
+  contentFeed: (hasComment) => ({
+    flex: hasComment ? 1 : 0,
     marginHorizontal: 6,
-    backgroundColor: COLORS.white
-  },
+    backgroundColor: COLORS.white,
+    marginBottom: hasComment ? 0 : normalizeFontSizeByWidth(16)
+  }),
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The new styles for `contentFeed` are based on the `hasComment` prop. This is a good use of conditional styling. However, it's important to note that if `hasComment` is `undefined`, it will be treated as `false`. If this is not the intended behavior, you should ensure that `hasComment` always has a boolean value.
```diff
-  contentFeed: (hasComment) => ({
-    flex: hasComment ? 1 : 0,
-    marginHorizontal: 6,
-    backgroundColor: COLORS.white,
-    marginBottom: hasComment ? 0 : normalizeFontSizeByWidth(16)
-  }),
+  contentFeed: (hasComment = false) => ({
+    flex: hasComment ? 1 : 0,
+    marginHorizontal: 6,
+    backgroundColor: COLORS.white,
+    marginBottom: hasComment ? 0 : normalizeFontSizeByWidth(16)
+  }),
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->